lua/nvim-treesitter/ts_utils.lua

```diff
@@ -143,7 +143,7 @@ function M.get_node_at_cursor(winnr)
   return root:named_descendant_for_range(cursor_range[1], cursor_range[2], cursor_range[1], cursor_range[2])
 end
 
-function M.get_root_for_position(line, col, root_lang_tree)
+function M.get_root_for_position(line, col, root_lang_tree, exclude_child_trees)
```

Comment:
I think this function is not necessary. When you specify the `root_lang_tree` you already have it and there's no need to call the function and calling this function with  `line, col, nil, true` feels a bit weird.

Reply:
fair point, removed this arg